### ISSUE TITLE
Cannot find DIA-NN search outputs

### ISSUE THREAD

**Issue by ClT4** [vdemichev/DiaNN#548]
Hi I am running a DIA-NN search on 154 .dia files. I have generated a fold for the results files to go into and indicated the file path in the 'main output' box. There are no errors reported but the run seems to stop at file 60/154. I then get the message below, but the indicated folder is empty when I go to check for the files.

[200:47] File #60/154
[200:47] Loading run F:\Abbott_MS\2021_09_21_CT_G04_S1-G4_1_1678.d

DIA-NN exited
DIA-NN-plotter.exe "F:\Abbott_MS\DIA_NN_lrg_lib_search\Abbott_search.stats.tsv" "F:\Abbott_MS\DIA_NN_lrg_lib_search\Abbott_search.tsv" "F:\Abbott_MS\DIA_NN_lrg_lib_search\Abbott_search.pdf"
PDF report will be generated in the background
![DIA nn settings](https://user-images.githubusercontent.com/89473429/199706907-9a4d3cb8-7cf1-4420-b617-846d5010f5fd.png)

**Comment by vdemichev** [comment]
Something wrong with that run... Exclude it?

**Comment by ClT4** [comment]
Yes I tried deleting this file but it stopped again at file 60/153, even though this is now a different file. I guess there could be something wrong with a number of files...?
